tools/nodered/scripts/setup.sh

```diff
@@ -1,3 +1,24 @@
-#!/usr/bin/bash
+#!/usr/bin/env bash
 source $(t3x -E)
 
+SCRIPT_DIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" > /dev/null; pwd)
+# Define the path to the template settings file for Node-RED
+SETTINGS_TEMPLATE="$SCRIPT_DIR/../files/node-red-default-settings.js"
+# Define the path to the Node-RED settings file
+SETTINGS_FILE="$HOME/.node-red/settings.js"
+
+echo "running node red installer..."
+bash <(curl -sL https://raw.githubusercontent.com/node-red/linux-installers/master/deb/update-nodejs-and-nodered) --confirm-root --confirm-install --confirm-pi --no-init
```

Comment:
- Did we want to run a cached copy?
- Should there be something that detects if this has been done already and skip this step?